static/js/app.js

```diff
@@ -159,3 +164,50 @@ newsletterForm?.addEventListener('submit', (event) => {
       console.error('Error:', error);
     });
 });
+
+// submit ensign form
+
+const ensignForm = document.getElementById('ensignForm');
+
+ensignForm?.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const formData = new FormData(ensignForm);
+  const data = Object.fromEntries(formData);
+  const ensignHomeEl = document.getElementById('ensign-home');
+  const ensignConfirmationEl = document.getElementById('ensign-confirmation');
+  console.log('ensignConfirmationEl', ensignConfirmationEl);
+
+  const { notify_me, ...rest } = data;
+
+  const formattedData = {
+    notifications: !!(notify_me === 'on'),
+    ...rest,
+  };
+  console.log(formattedData);
+  fetch('https://api.rotationallabs.com/v1/notifications/signup', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(formattedData),
+  })
+    .then((response) => {
+      if (response.status === 204) {
+        ensignHomeEl.style.display = 'none';
+        ensignConfirmationEl.style.display = 'block';
+        ensignForm.reset();
+      }
+      return response.json();
+    })
+    .then((data) => {
+      console.log('Success:', data);
+    })
+
+    .catch((error) => {
+      console.error('Error:', error);
+      ensignForm.reset();
+      ensignHomeEl.style.display = 'none';
+      ensignConfirmationEl.style.display = 'block';
+      footerBackground.style.backgroundColor = '#FFFFFF'
```

Comment:
This changes the footer background color back to white once the Ensign confirmation page appears.

Reply:
Would you mind removing that script in the catch block ? it was just for a test purpose. you can add the background update after the line 197

Reply:
I put that wrong spot by mistake. Thanks for catching that! The script has been removed from the catch block.